### ISSUE TITLE
I am not sure how to use Streamwall 

### ISSUE THREAD

**Issue by amarmashruwala** [streamwall/streamwall#205]
Hi There, i do not know how to use streamwall. Can anyone please please help!!

attached is what i see. What am i doing wrong or what am i suppose to do to watch https:// videos 

![Capture](https://github.com/user-attachments/assets/de8788ab-ce0b-4be5-9b44-d0e8d40fb25c)

**Comment by sean1138** [comment]
wish i could, looks like you were able to get a stream under "live" in the gui at least

<img width="1735" height="1017" alt="Image" src="https://github.com/user-attachments/assets/aabd2c56-d274-4b70-8b9d-d256bc3de99c" />

**Comment by amarmashruwala** [comment]
Yes i dont recall exactly what i did - the gui is a little clunky. but i will need to take a look again.

**Comment by sean1138** [comment]
<img width="1923" height="1083" alt="Image" src="https://github.com/user-attachments/assets/67d4d38e-7f8b-46be-bba5-0043920fa274" />

figured it out somewhat:
1 click a grid cell on left
2, click offline/unknown stream label at right
the video stream should now show in the grid cell you clicked first.

it's saving and reading from a config.toml file in ``C:\Users\username\AppData\Roaming\Streamwall\` which i would rather it not.  i've tried variations of 
```
npm -w packages/streamwall start --config-"../_config.toml"
npm -w packages/streamwall start --config-"./_config.toml"
npm -w packages/streamwall start --config-"_config.toml"
```
 to get it to read from my own config file but no luck yet.

i found if i click and drag across grid cells that will reduce the number of grid cells and make a stream fill multipl cells that i dragged across.  

is there a way to add grid cells without editing the config.toml file and restarting streamwall?
is there a way to launch it so i can move the window with video streams around and it's not stuck in the middle of my main monitor?
